.github/workflows/clean-assets.yml

```diff
@@ -1,4 +1,4 @@
-name: Handle new assets
+name: autofix.ci
```

Comment:
```suggestion
name: Clean new assets
```

Reply:
This is necessary for security reasons, see here: https://autofix.ci/security